test/suites/integration/user/file-upload-storage-request.test.ts

```diff
@@ -0,0 +1,466 @@
+      strictEqual(dataBlob.peerIds.length, 1);
+      strictEqual(dataBlob.peerIds[0].toHuman(), NODE_INFOS.user.expectedPeerId);
+
+      const issueStorageRequestResultTwice = await user_api.sealBlock(
+        user_api.tx.fileSystem.issueStorageRequest(
+          newBucketEventDataBlob.bucketId,
+          destination,
+          TEST_ARTEFACTS["res/smile.jpg"].fingerprint,
+          TEST_ARTEFACTS["res/smile.jpg"].size,
+          DUMMY_MSP_ID,
+          [NODE_INFOS.user.expectedPeerId]
+        ),
+        shUser
+      );
+
+      await sleep(500);
+
+      strictEqual(issueStorageRequestResultTwice.extSuccess, false);
+    });
+  });
+}
+
+for (const bspNetConfig of bspNetConfigCases) {
+  describe("User: Load File Into Storage", () => {
+    let user_api: BspNetApi;
+    let bsp_api: BspNetApi;
+
+    before(async () => {
+      await runBspNet(bspNetConfig);
+      user_api = await createApiObject(`ws://127.0.0.1:${NODE_INFOS.user.port}`);
+      bsp_api = await createApiObject(`ws://127.0.0.1:${NODE_INFOS.bsp.port}`);
+    });
+
+    after(async () => {
+      await user_api.disconnect();
+      await bsp_api.disconnect();
+      await closeBspNet();
+    });
+
+    it("loadFileInStorage works", async () => {
+      const source = "res/adolphus.jpg";
+      const destination = "test/adolphus.jpg";
+      const bucketName = "bucket-0";
+
+      const newBucketEventEvent = await user_api.createBucket(bucketName);
+      const newBucketEventDataBlob =
+        user_api.events.fileSystem.NewBucket.is(newBucketEventEvent) && newBucketEventEvent.data;
+
+      if (!newBucketEventDataBlob) {
+        throw new Error("Event doesn't match Type");
+      }
+
+      const { location, fingerprint, file_size } =
+        await user_api.rpc.storagehubclient.loadFileInStorage(
+          source,
+          destination,
+          NODE_INFOS.user.AddressId,
+          newBucketEventDataBlob.bucketId
+        );
+
+      strictEqual(location.toHuman(), destination);
+      strictEqual(fingerprint.toString(), TEST_ARTEFACTS[source].fingerprint);
+      strictEqual(file_size.toBigInt(), TEST_ARTEFACTS[source].size);
+    });
+
+    it("loadFileInStorage fails if file is empty", async () => {
+      const source = "res/empty-file";
+      const destination = "test/empty-file";
+      const bucketName = "bucket-1";
+
+      const newBucketEventEvent = await user_api.createBucket(bucketName);
+      const newBucketEventDataBlob =
+        user_api.events.fileSystem.NewBucket.is(newBucketEventEvent) && newBucketEventEvent.data;
+
+      if (!newBucketEventDataBlob) {
+        throw new Error("Event doesn't match Type");
+      }
+
+      try {
+        await user_api.rpc.storagehubclient.loadFileInStorage(
+          source,
+          destination,
+          NODE_INFOS.user.AddressId,
+          newBucketEventDataBlob.bucketId
+        );
+      } catch (e) {
+        strictEqual(e.message, "-32603: Internal error: FileIsEmpty");
```

Comment:
TypeScript is complaining here because `e`'s type is unknown.

Reply:
Thanks, I added the type `any` to the errors, since we are only interested in checking their message and not on the type itself

Reply:
[reference](https://www.typescriptlang.org/play/?ts=4.0.2#code/PTAECEFMGMEMFcDOlQClYDdYGVoCcBLABwBdRYAbCgewHdFQSALPOggOwHNz2BPULBXiQANKAAqvIpFyFSAKBCgAJtUgN21MonhEi1PGWUwKsQl0ZMUJKSmoAzHqEh5WeefJJ5+Ab3mhQJQA6IPkAX1A4EmgmUAAKSABKUB9QMI8lAAkCRBIDAjgqXjFmHNAmWAYAW0hYdjJmWAarUAADSFbQDksUKJjFMFpqeAplFUh7BAoG6h5eIIkmMsoaWkgx+wMe0Hs8SHXqKsZqAfI8ACMCLzMCCn5YaGh1Bjr+IlZpQ3nQAHUr2IALEEAAxiNYAcgwKBo1GQ7HWp2YKD2uUI0BIBGo7AYWMYtnIiEQBE47Bq9S67G2fVi0FMSEgiNmKzooHOWlirVenTqY1a8HYAGtNLR2K1QqdsLAaqzIBUMJi8KBaP85gAuTzeFL+QJgELhSJNGLxJJagKQIK5B4CgDc4QyYAAogAPIgUApXGVyhVKlX8oV0djqjW+bXBUIRanG1WgP3C9jJPwBJQATWGBvYmjI9LaHXIZBWMfqt22NmkobAiF4uUgRwotTw2KVFTIHpyYk2iqqBhQHE2oGQkHVSbAkCdUtdg9jAYAtDzp-Coe4zRaSFbbdqCI4EhTLewng5QNhePVYE6Ha4DAntcvLdAbdr0mEgA)